src/cascadia/Remoting/Monarch.h

```diff
@@ -67,6 +67,7 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
 
         winrt::Microsoft::Terminal::Remoting::IPeasant _getPeasant(uint64_t peasantID);
         uint64_t _getMostRecentPeasantID(bool limitToCurrentDesktop);
+        uint64_t _lookupPeasantIdForName(const winrt::hstring& name);

```

Comment:
should this be a `wstring_view` to make it more broadly applicable? hstring implicitly converts to wstring_view as necessary.